Production_scripts/Process_extracts/extract_care_home.R

```diff
@@ -0,0 +1,123 @@
+#####################################################
+# Care Home Extract
+# Author: Catherine Holland
+# Date: February 2022
+# Written on RStudio Server
+# Version of R - 3.6.1
+# Input - Social Care Care Home Episodes
+# Description - Match on Client data and Costs data
+#####################################################
+
+
+# Load packages
+library(dplyr)
+library(createslf)
+library(phsmethods)
+library(lubridate)
+
+
+fyyear <- check_year_format("1920")
```

Comment:
Use `year` instead of `fyyear` to be consistent with other scripts (or change all other scripts to be consistent 😜)

Reply:
the issue I was finding was when the year column is created within the script further down ~line 57 I want to use the year to  compute end_fy but it takes the year variable rather than vector, (these are in different formats) which is why I changed the name

can try rearranging to counteract this

Reply:
fixed as year was being entered in wrong format into the data